docs/reference/enable-gpu.rst

```diff
@@ -0,0 +1,32 @@
+Enable Nvidia GPUs
+==================
+
+To be able to run workloads on GPU with DSS, you need to have NVIDIA drivers installed on your machine. This can be done using `MicroK8s GPU add-on`_.
+
+Before installing DSS:
+
+1. Follow the MicroK8s documentation to `Install MicroK8s`_.
+
+2. Enable MicroK8s add-ons, including GPU.
+
+    .. code-block:: bash
+
+        microk8s enable storage dns rbac gpu
+
+3. Wait for GPU operator components to be ready. This can take around 5 minutes.
+
+    .. code-block:: bash
+
+        while ! sudo microk8s.kubectl logs -n gpu-operator-resources -l app=nvidia-operator-validator | grep "all validations are successful"
+        do
+          echo "waiting for validations"
+          sleep 5
+        done
+
+4. Install DSS snap and initialise.
+
+    .. code-block:: bash
+ 
+        microk8s config > ~/.dss/config
+        sudo snap install data-science-stack
+        dss initialize
```

Comment:
We can also add that you acn check everitime the GPU status in the cluster running `dss status`

This is the output before initialization 
```
[INFO] MLflow deployment: Not ready
[INFO] GPU acceleration: Enabled (NVIDIA-GeForce-RTX-3070-Ti)
```

this is after 

```
[INFO] MLflow deployment: Ready
[INFO] MLflow URL: http://10.152.183.105:5000
[INFO] GPU acceleration: Enabled (NVIDIA-GeForce-RTX-3070-Ti)
```

note the gpu model will be different depending on card

Reply:
good suggestion
on this note I feel this doc is becoming more of a how-to not a reference, but I'm conflicted because it's not part of the core dss
what do you guys think?